pkg/xds/bootstrap/template_v3_test.go

```diff
@@ -0,0 +1,90 @@
+package bootstrap
+
+import (
+	"errors"
+	"net"
+
+	envoy_cluster_v3 "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("dnsLookupFamilyFromXdsHost", func() {
```

Comment:
nit: very often when we are doing same action with differnt input expecting differnt output, we use DescribeTable. Not saying you should rewrite this test, because I'm not sure if it make sense in this context. Just keep an eye on it.